sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -43,18 +43,17 @@ sealed trait TimestampFormatter extends Serializable {
 }
 
 class Iso8601TimestampFormatter(
-    pattern: String,
-    timeZone: TimeZone,
-    locale: Locale) extends TimestampFormatter with DateTimeFormatterHelper {
+     pattern: String,
```

Comment:
nit: indent

Reply:
thanx. Just wondering why `scalastyle` didn't catch that.